### DISCUSSION TITLE
Providing options for LazyVim/LazyVim?

### DISCUSSION THREAD

**Discussion by KoinBanks** [LazyVim/LazyVim#3957] [Q&A] [answered]
Hey, I wanted to add the "Variable" syntax kind to `LazyVim.options.kind_filter.default` list so that `<leader>ss` will also display variables.

But how exactly can I do this? 

I've tried this in my plugins folder but to no avail as the opts are empty at this point.

```lua
  {
    "LazyVim/LazyVim",
    opts = function(_, opts)
      print(vim.inspect(opts)) -- this will print empty object
    end,
  },
```

Thanks for answer.

**Comment by dpetka2001** [comment, marked as answer]
Try [this answer here](https://github.com/LazyVim/LazyVim/issues/2216#issuecomment-2091460516).

**Reply by KoinBanks** [reply]
Huh, so it works - it just can't accept the customizer.

Thanks for help!
